source/beamline/AMSlit.cpp

```diff
@@ -213,12 +212,13 @@ void AMSlit::setSecondBlade(AMControl *newControl, BladeOrientation orientation)
 	}
 }
 
-void AMSlit::removeSecondBlade()
+bool AMSlit::removeSecondBlade()
```

Comment:
maybe this can be combined with `removeFirstBlade()` and reimplemented as `removeBlade(*blade)`